src/esw/brushless.py

```diff
@@ -234,8 +325,52 @@ def __init__(self):
         else:
             transport = None
 
-        for name, info in drive_info_by_name.items():
-            self.drive_bridge_by_name[name] = MoteusBridge(info["id"], transport)
+        arm_controller_info_by_name = rospy.get_param("brushless/arm/controllers")
+
+        self._max_rps_by_name = {}
+        for name, info in arm_controller_info_by_name.items():
+            self._max_rps_by_name[name] = info["max_rps"]
+
+        self._motors_manager = MotorsManager(arm_controller_info_by_name, ArmManager.ARM_NAMES, transport, "arm_status")
+        rospy.Subscriber("ra_cmd", JointState, self._process_ra_cmd)
+
+    def _process_ra_cmd(self, ros_msg: JointState) -> None:
+        """
+        Process a JointState command controls individual motor speeds.
+        :param ros_msg: Has information to control the joints controlled by both brushed and brushless motors.
+        We only care about the joints controlled by brushless motors.
+        """
+        self._motors_manager.update_time()
+
+        for i, name in enumerate(ros_msg.name):
+            if name in ArmManager.ARM_NAMES:
+                position, velocity, torque = (
+                    ros_msg.position[i],
+                    ros_msg.velocity[i],
+                    ros_msg.effort[i],
+                )
+
+                # We usually assume that the ros_msg sends velocity commands from -1 to 1,
+                # but change it just in case.
+                if abs(velocity) < 1:
```

Comment:
Uh... I think this is supposed to be greater than, not less than?

Reply:
you're right, I made changes